model-optimizer/extensions/front/InterpolateV1ToInterpolate.py

```diff
@@ -0,0 +1,61 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from extensions.ops.interpolate import Interpolate
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.graph.graph import Graph
+from mo.ops.const import Const
+
+
+def correct_pad(pad):
+    return int64_array([pad] if not isinstance(pad, list) else pad)
+
+
+class InterpolateV1ToInterpolate(FrontReplacementPattern):
```

Comment:
@vgavrilo, @lazarevevgeny, why do we need this transformation if nGraph should upgrade a version? This transformation looks strange to me.

Reply:
The corresponding nGraph transformation is only to read old IRs.